### DISCUSSION TITLE
How to get the number of requests from all EC2 instances?

### DISCUSSION THREAD

**Discussion by Rockyread** [community/community#63955] [Programming Help] [answered]
### Body

I have several EC2 instances and running my python project on them.
What I wanna do is just monitor the number of requests from EC2 instances which they get from the users by real time.
And I want to write this in python code.
I have tried a lot of things, but can't solve this problem.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mehulrajdev** [comment, marked as answer]
This might help
https://saturncloud.io/blog/how-to-see-all-running-amazon-ec2-instances-across-all-regions/


To request access to this feature:

1  Open [AWS Support Center](https://console.aws.amazon.com/support/home#/).

2  Choose to Create case.

3  Choose Account and billing.

4  For Service, choose General Info and Getting Started.

5  For Category, choose Using AWS & Services.

6  Choose the Next step: Additional information.

7  For Subject, enter Request access to Amazon EC2 API metrics.

8  For Description, enter Please grant my account access to Amazon EC2 API metrics. Related page: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/monitor.html. Also, include the Region where you need access.

9  Choose the Next step: Solve now or contact us.

10  On the Contact Us tab, choose your preferred contact language and method of contact.

Choose Submit.
